packages/react/src/primitives/Breadcrumbs/BreadcrumbLink.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import classNames from 'classnames';
+
+import { classNameModifierByFlag } from '../shared/utils';
+import {
+  BreadcrumbsLinkProps,
+  ForwardRefPrimitive,
+  Primitive,
+  BaseBreadcrumbLinkProps,
+} from '../types';
+import { ComponentClassNames } from '../shared/constants';
+import { Link } from '../Link';
+import { Text } from '../Text';
+import { BreadcrumbsContext } from './BreadcrumbsContext';
+
+const BreadcrumbLinkPrimitive: Primitive<BreadcrumbsLinkProps, 'a'> = (
+  { className, children, href, ...rest },
+  ref
+) => {
+  const { isCurrent } = React.useContext(BreadcrumbsContext);
+  const componentClasses = classNames(
+    ComponentClassNames.BreadcrumbsLink,
+    classNameModifierByFlag(
+      ComponentClassNames.BreadcrumbsLink,
+      'current',
+      isCurrent
+    ),
+    className
+  );
+
+  if (isCurrent) {
+    return (
+      <Text
+        as="span"
+        aria-current="page"
+        className={componentClasses}
+        ref={ref}
+        {...rest}
```

Comment:
If the types we allow for `BreadcrumbLinkPrimitive` are for an `anchor` element, any concerns with passing them to a `Text` element?